.repolite.yml

```diff
@@ -0,0 +1,31 @@
+prog_name: repolite
+top_dir: ext  # local directory path for enabled repositories
+pull_with_rebase: false  # use --ff-only if false
+# add new repo_name sections as needed
+repos:
+  - repo_name: py3tftp
```

Comment:
Think this is not used, remove?

Reply:
Actually I put both packages there for test/comparison as needed.  As of now tftpy looks okay using the last release from pypi so no need for syncing repositories just yet.